### ISSUE TITLE
Add explainers on the activity page

### ISSUE THREAD

**Issue by win7guru** [butzist/ActivityLauncher#245]
Explainers in this case are short descriptions for what an activity does, in laymans terms.  I have tried other apps that expose activities and they also have one or two words describing the activities.  The problem that I have is that there are multiple activities that have the same one or two word description.  Sure, the long name that has all the parameters is there, but it would be much easier if it said something like

...
> Sign in to app
or
> Sign in as name
or
> Open new tab
...

My suggestion is to either add a context menu entry that can show a popup/popover or a way to tap a little information overlay to achieve the same thing.

Let me know what you think.

![Screenshot_20240226_231224_Activity Launcher.jpg](https://github.com/butzist/ActivityLauncher/assets/650640/2aac5e64-56ff-4b30-a4f9-e80ac385381a)

**Comment by drogga** [comment]
@win7guru wrote: "I have tried other apps that expose activities and they also have one or two words describing the activities."

Like...?
I hope you don't confuse this with permissions descriptions, because if an activity has a label, then it will be shown (it's supposed to). Consider trying out v2.0.0 (you can DL it from the Actions tab or F-Droid, IDK if it's available as Beta on PS yet).
